src/main/java/com/develhope/spring/users/models/User.java

```diff
@@ -2,17 +2,25 @@
 
 import com.develhope.spring.users.models.exceptions.EmptyParameterException;
 import com.develhope.spring.users.models.exceptions.WrongEmailFormatException;
+import jakarta.persistence.*;
 
+@Entity
+@Table(name = "users")
 public class User {
 
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private long id;
+    @Column(nullable = false)
     private String name;
-
+    @Column(nullable = false)
     private String surname;
 
     private long phoneNumber;
```

Comment:
Secondo me phoneNumber pure dovrebbe essere messo come column,  anche se può essere nullo

Reply:
di base se non segni @Column sono comunque colonne. L'annotazione la metti solo se vuoi specificare qualcosa. Vale lo stesso discorso di @Table che se lo metti ma senza specificare nulla è superfluo. Non l'ho specificato in alcuni casi poichè boh ho pensato che alcuni valori potesse appunto essere nulli e quindi non sto a specificare null = true dato che se non lo metti è implicito. Però se volete per chiarezza o altro lo inserisco